src/order/conditionally_complete_lattice.lean

```diff
@@ -463,6 +463,11 @@ begin
   { exact csupr_le (λ x, le_csupr_of_le B x (H x)) },
 end
 
+lemma le_csupr_set {f : β → α} {s : set β}
+  (H : bdd_above (f '' s)) {c : β} (hc : c ∈ s) : f c ≤ ⨆ i : s, f i :=
+le_csupr (by simp_rw [range_comp f, subtype.range_coe, H] : bdd_above $ range (λ i : s, f i))
+  ⟨c, hc⟩
```

Comment:
Golfs to:
```suggestion
(le_cSup H ⟨_, hc, rfl⟩).trans_eq Sup_image'
```
Are we sure we need this  lemma, given you can get here by doing `rw ←Sup_image'`? Perhaps a more useful thing to do would be to alias `←Sup_image'` to `supr_set_subtype` or similar.

Reply:
This is a much better proof. I think it's nice to be able to spell things with `⨆`, so having some API for that is probably nice.